includes/admin/class-wc-admin-notices.php

```diff
@@ -428,6 +430,38 @@ public static function wp_php_min_requirements_notice() {
 		include dirname( __FILE__ ) . '/views/html-notice-wp-php-minimum-requirements.php';
 	}
 
+	/**
+	 * Add MaxMind MIssing license key notice.
+	 *
+	 * @since 3.9.0
+	 */
+	public static function add_maxmind_missing_license_key_notice() {
+		$default_address = get_option( 'woocommerce_default_customer_address' );
+
+		if ( ! in_array( $default_address, array( 'geolocation', 'geolocation_ajax' ), true ) ) {
+			return;
+		}
+
+		$integration_options = get_option( 'woocommerce_maxmind_geolocation_settings' );
+		if ( empty( $integration_options['license_key'] ) ) {
+			self::add_notice( 'maxmind_license_key' );
+		}
+	}
+
+	/**
+	 * Display MaxMind Missing license key notice.
+	 *
+	 * @since 3.9.0
+	 */
+	public static function maxmind_missing_license_key_notice() {
+		if ( get_user_meta( get_current_user_id(), 'dismissed_maxmind_license_key_notice', true ) ) {
```

Comment:
also add clause for `woocommerce_maxmind_geolocation_display_notices` filter.

Reply:
Good catch!